src/console/ControllerTrait.php

```diff
@@ -79,4 +80,28 @@ protected function outputCommand(string $command, bool $withScriptName = true)
     {
         Console::outputCommand($command, $withScriptName);
     }
+
+    /**
+     * Creates a function for the `validator` option of `Controller::prompt`.
+     *
+     * @param Model $model
+     * @param string $attribute
+     * @param string|null $error
+     * @return callable
+     */
+    protected function createInputValidator(Model $model, string $attribute, ?string &$error = null): callable
```

Comment:
@timkelty Looks like this `$error` attribute is unused by the method?

Reply:
Ah, yeah. That was to satisfy the nested pass by ref weirdness, but wasn't very clear or needed. Updated PR.